auth/routes.js

```diff
@@ -83,12 +84,14 @@ module.exports = function(router) {
     })
     // create new projects on the database
     .put(function(req, res) {
```

Comment:
The changes to routes.js undo the latest changes I made to the file. What do you want to do here, revert them and then have me redo the PR. I've still got the files, otherwise you can update them in your branch from dev.

Reply:
What happened is `bugs/dropdown` PR undid all of my changes in the PR before that (React code refactor). So what I did was to add my changes back in, then looked through the changes you made and add those in afterwards. Looks like some of your latest changes were made on top of old code, so in some places I had to code a bit differently than you did but functionally what you changed should still be there. 

If I missed any of your changes, I'm sorry about that! My intention isn't to undo your changes. If you let me know I'm happy to add those back in!

Reply:
Since i'm the one who caused most of this I think I should probably leave it and redo it once your code is approved, since it's just one file.